### ISSUE TITLE
Create dll file

### ISSUE THREAD

**Issue by Zumbalamambo** [b-com-software-basis/xpcf#1]
How do I create a dll file of xpcf?

**Comment by Firefly35** [comment]
A new 2.3.1 release is on the go in the next days. I suggest you use the qt creator wizards provided with xpcf to create an xpcf module with your own components.
To build xpcf, you'll need to install pkg-config on windows and conan (you can follow the How to build part of the readme).
Let me know if any information is missing.

**Comment by Zumbalamambo** [comment]
@Firefly35  Qt throws initialization errors while trying to open xpcf.pro. is there any prebuild binaries?

**Comment by Firefly35** [comment]
Yes indeed, xpcf binaries are available in the SolAR framework context here : https://github.com/SolarFramework/binaries/releases

I'm interested to figure out the problems you had opening xpcf.pro, can you give me more feedback/logs about it please ?

**Comment by Zumbalamambo** [comment]
@Firefly35 I have managed to open xpcf.pro with QTCreator. While configuring the project, I'm getting the following error.

`builddefs/qmake/templatelibconfig.pri: No such file or directory`

**Comment by Firefly35** [comment]
You need to clone xpcf repository with sub modules recursively. It will find the building rules afterwardsEnvoyé depuis mon smartphone Samsung Galaxy.
-------- Message d'origine --------De : Zumbalamambo <notifications@github.com> Date : 15/02/2020  18:21  (GMT+01:00) À : b-com-software-basis/xpcf <xpcf@noreply.github.com> Cc : Loïc Touraine <loic.touraine@gmail.com>, Mention <mention@noreply.github.com> Objet : Re: [b-com-software-basis/xpcf] Create dll file (#1) @Firefly35 I have managed to open xpcf.pro with QTCreator. While configuring the project, I'm getting the following error.
builddefs/qmake/templatelibconfig.pri: No such file or directory

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/b-com-software-basis/xpcf/issues/1?email_source=notifications\u0026email_token=AC5Z2FA23WTVNJGBZ3KFRXLRDAP77A5CNFSM4KU2PGTKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEL3SIVI#issuecomment-586622037",
"url": "https://github.com/b-com-software-basis/xpcf/issues/1?email_source=notifications\u0026email_token=AC5Z2FA23WTVNJGBZ3KFRXLRDAP77A5CNFSM4KU2PGTKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEL3SIVI#issuecomment-586622037",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Firefly35** [comment]
Hello, did you succeed to build xpcf ?
